tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
```YAML
# Default, access with root without sudo
ansible_ssh_user: root
ansible_become: false
```

and there we go :wink:

I guess with those settings, other things will break as well. Have you checked out [debops.bootstrap](https://github.com/debops/ansible-bootstrap)?